ql/src/test/queries/clientpositive/list_bucket_dml_8.q

```diff
@@ -1,11 +1,10 @@
 --! qt:dataset:srcpart
 set hive.mapred.mode=nonstrict;
 set hive.exec.dynamic.partition=true;
-set hive.input.format=org.apache.hadoop.hive.ql.io.BucketizedHiveInputFormat;
+set hive.tez.input.format=org.apache.hadoop.hive.ql.io.BucketizedHiveInputFormat;
 set hive.merge.smallfiles.avgsize=200;
 set mapred.input.dir.recursive=true;
-set hive.merge.mapfiles=false;	
-set hive.merge.mapredfiles=false;
+set hive.merge.tezfiles=false;
```

Comment:
So if we use MR it wont work?

Reply:
Yes. This configuration lets Hive create multiple small files and prevents merging them until `ALTER ... CONCATENATE` command. So running the modified qfile with MR produces wrong result.